tools/ramdump/ramdump_tool.c

```diff
@@ -389,13 +452,26 @@ int main(int argc, char *argv[])
 		goto tty_lock_err;
 	}
 
-	if (do_handshake(dev_fd) != 0) {
+	printf("Target device locked and ready to DUMP!!!\n");
+
+	printf("Would you like to dump RAM regions?(y/Y = \'YES\' || any other character = \'NO\')\n");
```

Comment:
How about giving the list?
```
1. RAMDUMP
2. FSDUMP
3. ALL or Both
```

Reply:
Could you check the reply on crashdump.c, Line 189 please?
Currently the user does have a choice amongst dumping none, either one or both partitions.
Would you prefer a list format for choices?

Reply:
1. User should wait and give the feedback after sometime for fs dump. I think it's inconvenient. 
    If we provide the list, user can give the input and no need to concern.
2. You know the procedure so that you know fsdump can be done after ramdump.
    But normal user couldn't. At the start of dump, he could get confused for no option for fs dump. 
3. When you want to get fsdump only, no way to do. You should get unnecessary RAMDUMP first and then can get fsdump.

Reply:
Oh you provide the `no` option. Please ignore my 3rd comment.

Reply:
> Oh you provide the no option. Please ignore my 3rd comment.

3rd comment means `3. When you want to get fsdump only, no way to do. You should get unnecessary RAMDUMP first and then can get fsdump.`. Please let me know your opinion for my 1st and 2nd.

Reply:
Keeping in mind points 1 and 2, list makes the process easy.
I updated how the choice is made by the user.